### PR TITLE
Update docker script to allow for configuration of virtualhost

### PR DESCRIPTION
**Pull request by hydrogen18**

### CODE REVIEW

**Comment by andrewgaul** [comment]
Thank you for your contribution @hydrogen18!

File: src/main/resources/run-docker-container.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 exec java \
     -DLOG_LEVEL=${LOG_LEVEL} \
     -Ds3proxy.endpoint=http://0.0.0.0:80 \
+    -Ds3proxy.virtual-host=${S3PROXY_VIRTUALHOST} \
```
Comment (jim-slattery-rs) [inline src/main/resources/run-docker-container.sh]: our builds using the s3proxy dockerhub image started breaking when this change was introduced..

maybe `virtual-host` used to be null? is there any way for it to still be null if it's included in the command line like this?

Comment (hydrogen18) [inline src/main/resources/run-docker-container.sh]: Perhaps we just need to have the configuration methods consider `null` and the empty string to be the same thing.

**Comment by andrewgaul** [comment]
@jim-slattery-rs 6b30e6612779e54c1741e522a09d6bd6b2dd31b4 should fix this and Travis updated the Docker image.  Sorry about the breakage and thanks for your feedback!

File: src/main/resources/run-docker-container.sh (hunk 1 above)
Comment (andrewgaul) [inline src/main/resources/run-docker-container.sh]: @jim-slattery-rs 6b30e6612779e54c1741e522a09d6bd6b2dd31b4 should fix this and Travis updated the Docker image.  Sorry about the breakage and thanks for your feedback!

**Pull request merged**